src/main/java/com/stripe/model/Plan.java

```diff
@@ -21,8 +21,10 @@
 	Boolean livemode;
 	Map<String, String> metadata;
 	String name;
```

Comment:
Can you move the `name`, `statementDescriptor` and `trialPeriodDays` attributes to the bottom of the list with a `@Deprecated` tag?